add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    $title = $_POST['title'];
```

Comment:
Observations: The current implementation does not have any error handling for invalid inputs or empty fields. Additionally, there is no validation for the year input to ensure it is a valid number.

Recommendations: Implement server-side validation to ensure all required fields are filled and the year is a valid number. Provide user-friendly error messages.